python/tvm/relay/build_module.py

```diff
@@ -376,10 +381,11 @@ def build(ir_mod, target=None, target_host=None, params=None, mod_name="default"
         )
         func_metadata = bld_mod.get_function_metadata()
         devices = bld_mod.get_devices()
+        final_ir_mods = bld_mod.get_irmodule()
 
         if executor == "aot":
             executor_factory = _executor_factory.AOTExecutorFactoryModule(
-                ir_mod, target, runtime_mod, mod_name, params, func_metadata, devices
+                ir_mod, final_ir_mods, target, runtime_mod, mod_name, params, func_metadata, devices
```

Comment:
We should use a better term and possibly change the other ir_mod to disambiguate, though Im not sure what to call the second argument here.

Reply:
Yeah, ideally it'd all be one big finished `ir_mod`, happy for name suggestions :smile_cat:

Reply:
lowered_ir_mods ?